docs/ru/mkdocs.yml

```
@@ -54,6 +54,10 @@ nav:
   - tr: /tr/
   - uk: /uk/
   - zh: /zh/
+- python-types.md
+- Tutorial - User Guide:
```

```suggestion
- Учебник - руководство пользователя:
```

Nice! This would be the place to translate that text to Russian, I don't really know how that is properly written, I just used DeepL, it's probably incorrectly written, but you get the idea.